src/test/kotlin/uk/gov/justice/digital/hmpps/courtregister/config/PostgresContainer.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.courtregister.config
+
+import org.testcontainers.containers.PostgreSQLContainer
+import org.testcontainers.containers.wait.strategy.Wait
+import java.io.IOException
+import java.net.ServerSocket
+
+object PostgresContainer {
+  val instance: PostgreSQLContainer<Nothing>? by lazy { startPostgresqlContainer() }
```

Comment:
so I guess this assumes in Circle to postgres is already up, I wonder if we will have timing issues, feels like that we might need to wait for postgres to be up before proceeding to `check` step

Reply:
Hmmm it does yes.  Not seen that issue (yet!) and Testcontainers claims to wait for the database to be up... :thinking: